omod/src/main/resources/config.xml

```diff
@@ -40,12 +40,27 @@
 	</advice>
 	<!-- /AOP -->
 
+    <globalProperty>
+        <property>patientmatching.configDirectory</property>
+        <defaultValue>patientmatching</defaultValue>
+        <description>The location of the patient matching configuration folder, can be absolute or relative path
+        </description>
+    </globalProperty>
+
 	<globalProperty>
-		<property>patientmatching.linkConfigFile</property>
-		<description>This file can be found at doc/link_config.xml
+		<property>patientmatching.configFile</property>
+        <defaultValue>patientmatching/config.xml</defaultValue>
+		<description>The location of the patient matching configuration file, can be absolute or relative path
 		</description>
 	</globalProperty>
 
+    <globalProperty>
+        <property>patientmatching.serializationDirectory</property>
+        <defaultValue>patientmatching/serial</defaultValue>
+        <description>The location of the serialization folder to be used when matching patients, can be absolute or relative path
```

Comment:
Since we are eventually using `getDirectoryInApplicationDataDirectory`, how would an absolute path work out?

Reply:
Take a closer look at the code in getDirectoryInApplicationDataDirectory, it supports absolute paths.

Reply:
Removed default value from xml file, it is set inside the code anyways